applications/tari_merge_mining_proxy/src/block_template_data.rs

```diff
@@ -37,18 +60,43 @@ impl BlockTemplateRepository {
     }
 
     pub async fn get<T: AsRef<[u8]>>(&self, hash: T) -> Option<BlockTemplateData> {
+        trace!(
+            target: LOG_TARGET,
+            "Retrieving blocktemplate with merge mining hash: {:x?}",
+            hash.as_ref()
```

Comment:
Still not printing as hex:
```
2021-01-13 12:15:58.044820400 [tari_mm_proxy::xmrig] TRACE Retrieving blocktemplate with merge mining hash: 
[79, 60, 3e, f3, 82, b8, 1e, ab, c4, a9, 63, 4c, 3c, 38, ae, 3b, 7, 3e, 9e, 92, 36, 43, cc, ce, d4, 35, a6, 8f, 8a, c5, 8a, f1]
```

Reply:
See above comment, will sort it out for all